services/connectivity/src/service/connectivity.js

```diff
@@ -149,7 +149,8 @@ module.exports = (connectivityPersistence) => {
       const paConnDataInSE = rawData && rawData[0] ? rawData[0] : null;
       if (!paConnDataInSE) {
         throw new Error(
-          `Data for Current PA Connectivity In ${strategicEcosystem} doesn't exists in the selected area id and area type`,
+          `Data for Current PA Connectivity In ${strategicEcosystem}
```

Comment:
After reconfiguring the rules, leave this line as it was

Reply:
@erikasv After reconfiguring the rules, the maximum line error is still displayed.

Reply:
![Screenshot_20210713_113741](https://user-images.githubusercontent.com/72577339/125492932-d89306ce-37e9-40dd-9f00-a7f904843bca.png)